src/extensions/ContentDiff/helpers/simpleObjects.js

```diff
@@ -71,13 +69,19 @@ export const createSimpleObject = async (control, space, fields, locale) => {
     default:
       break;
   }
-  const value = fields[control.fieldId] && (fields[control.fieldId][locale] || fields[control.fieldId].getValue());
```

Comment:
This is actually where the issue was. The field in question was a boolean field, with a "false" value. This was causing this statement, `fields[control.fieldId][locale]` to return false and thus attempt to go to the next statement. Fixed by using lodash `has` and `get`.